radix-transactions/src/builder/transaction_builder.rs

```diff
@@ -359,7 +359,7 @@ impl PartialTransactionV2Builder {
     ) -> (SignedPartialTransactionV2, TransactionObjectNames) {
         let (transaction, names) = self.build_with_names();
         let validator = TransactionValidator::new_with_static_config_network_agnostic(
-            ValidationConfig::latest(),
+            TransactionValidationConfig::latest(),
```

Comment:
Yeah I could not compile `develop` because of this so I had to make these changes. Figured that perhaps some accidental renaming work got in early in `develop`.